docs/supported-libraries.md

```diff
@@ -17,111 +17,134 @@ or [contributing](../CONTRIBUTING.md).
+| [Spring WebFlux](https://docs.spring.io/spring/docs/current/javadoc-api/org/springframework/web/reactive/package-summary.html)              | 5.0+                          | [opentelemetry-spring-webflux-5.0](../instrumentation/spring/spring-webflux-5.0/library)                                                                                                                                                                                                                                                                                                |                                                                                        |
+| [Spymemcached](https://github.com/couchbase/spymemcached)                                                                                   | 2.12+                         | N/A                                                                                                                                                                                                                                                                                                                                                                                     |                                                                                        |
+| [Tomcat JDBC Pool](https://tomcat.apache.org/tomcat-7.0-doc/jdbc-pool.html)                                                                 | 8.5.0+                        | N/A                                                                                                                                                                                                                                                                                                                                                                                     |                                                                                        |
+| [Twilio](https://github.com/twilio/twilio-java)                                                                                             | 6.6+ (not including 8.x yet)  | N/A                                                                                                                                                                                                                                                                                                                                                                                     | none                                                                                   |
```

Comment:
Removed `TODO` entries from the "Semantic Conventions" column. Blank entries still need attention. "none" indicates that the analysis is complete and none of the semantic conventions are implemented.